features/cron.feature

```diff
@@ -123,7 +124,9 @@ Feature: Manage WP-Cron events and schedules
     When I run `wp cron event delete wp_cli_test_event_5`
     Then STDOUT should be:
       """
-      Success: Deleted 2 instances of the cron event 'wp_cli_test_event_5'.
+      Deleted the cron event 'wp_cli_test_event_5'
```

Comment:
This seems like it would just flood the output if you happen to delete a corn event that has hundreds of instances. Is there a reason why the output was changed here?

Reply:
Hi @schlessera, I've removed this log. The tests that used these log files to verify if they succeeded have been changed to look at the output of `wp cron event list` instead of the output of the command being run.